app/src/modules/settings/routes/presets/item.vue

```diff
@@ -153,446 +153,423 @@ type FormattedPreset = {
+			};
+		},
+	});
 
-			fetchPreset();
+	const search = computed<string | null>({
+		get() {
+			return values.value.search;
+		},
+		set(newSearch) {
+			edits.value = {
+				...edits.value,
+				search: newSearch,
+			};
+		},
+	});
 
-			return { loading, preset, fetchPreset };
+	return { edits, initialValues, values, layoutQuery, layoutOptions, hasEdits, updateFilters, search };
 
-			async function fetchPreset() {
-				if (props.id === '+') return;
+	function updateFilters(newFilter: Filter) {
+		edits.value = {
+			...edits.value,
+			filter: newFilter,
+		};
+	}
+}
 
-				loading.value = true;
+function usePreset() {
+	const loading = ref(false);
+	const preset = ref<Preset | null>(null);
 
-				try {
-					const response = await api.get(`/presets/${props.id}`);
+	fetchPreset();
 
-					preset.value = response.data.data;
-				} catch (err: any) {
-					unexpectedError(err);
-				} finally {
-					loading.value = false;
-				}
-			}
-		}
+	return { loading, preset, fetchPreset };
 
-		function useLinks() {
-			const backLink = computed(() => {
-				return `/settings/presets`;
-			});
+	async function fetchPreset() {
+		if (props.id === '+') return;
 
-			return { backLink };
+		loading.value = true;
+
+		try {
+			const response = await api.get(`/presets/${props.id}`);
+
+			preset.value = response.data.data;
+		} catch (err: any) {
+			unexpectedError(err);
+		} finally {
+			loading.value = false;
 		}
+	}
+}
 
-		function useForm() {
-			const systemCollectionWhiteList = ['directus_users', 'directus_files', 'directus_activity'];
-
-			const fields = computed(() => [
-				{
-					field: 'collection',
-					name: t('collection'),
-					type: 'string',
-					meta: {
-						interface: 'select-dropdown',
-						options: {
-							choices: collectionsStore.collections
-								.map((collection) => ({
-									text: collection.name,
-									value: collection.collection,
-								}))
-								.filter((option) => {
-									if (option.value.startsWith('directus_')) return systemCollectionWhiteList.includes(option.value);
-
-									return true;
-								}),
-						},
-						width: 'half',
-					},
-				},
-				{
-					field: 'scope',
-					name: t('scope'),
-					type: 'string',
-					meta: {
-						interface: 'system-scope',
-						width: 'half',
-					},
-				},
-				{
-					field: 'layout',
-					name: t('layout'),
-					type: 'string',
-					meta: {
-						interface: 'select-dropdown',
-						options: {
-							choices: layouts.value.map((layout) => ({
-								text: layout.name,
-								value: layout.id,
-							})),
-						},
-						width: 'half',
-					},
-				},
-				{
-					field: 'name',
-					name: t('name'),
-					type: 'string',
-					meta: {
-						interface: 'system-input-translated-string',
-						width: 'half',
-						options: {
-							placeholder: t('preset_name_placeholder'),
-						},
-					},
+function useLinks() {
+	const backLink = computed(() => {
+		return `/settings/presets`;
+	});
+
+	return { backLink };
```

Comment:
making this into a computed seems like overkill to me as it's never changed by the looks of it 🤔 or im missing something

Reply:
But then again it already existed and was just moved i see 😕

Reply:
I'll just clean that up even though it was already there before.